packages/website/src/examples/Table/withBlankslate.example.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import {compose} from 'redux';
+
+import {Badge, TableHOC, TableRowConnected, tableWithBlankSlate, tableWithNewPagination} from '@coveord/plasma-react';
+
+export default () => <WithBlankSlate />;
+
+const WithBlankSlate: React.FunctionComponent = () => (
+    <>
+        <TableComposed
+            id="blankSlate"
+            className="table"
+            data={{} as any}
```

Comment:
Shouldn't we use an array instead?
```suggestion
            data={[]}
```